chapter2/tests/t19/terraform/_modules/nginx/variables.tf

```diff
@@ -0,0 +1,54 @@
+variable "container_resource" {
```

Comment:
not semantic name for the var. description is good - the name is bad.
`container_name` would be a lot more readable

Reply:
> not semantic name for the var. description is good - the name is bad. `container_name` would be a lot more readable

@dmitry-mightydevops 
Done